src/Orckestra.Composer.MyAccount/Services/MembershipViewService.cs

```diff
@@ -189,7 +189,8 @@ public virtual async Task<LoginViewModel> LoginAsync(LoginParam param)
 
             if (loginResponse)
             {
-                var user = Membership.GetUser(userName, true);
+                var profileSettings = await CustomerSettings.GetProfileSettingsAsync().ConfigureAwait(false);
+                var user = profileSettings.UseEmailAsUsername ? Membership.GetUserByEmail(userName, true) : Membership.GetUser(userName, true);
```

Comment:
we discussed with Dmytri, we do too many request to load user
Could you please try change Login API on platform to return also CustomerId.
so we can skip here having profileSettings and try have less calls to get user